test/unit/sidekiq_mover_test.rb

```diff
@@ -0,0 +1,81 @@
+require 'test_helper'
+# frozen_string_literal: true
+class TransferJobs::SidekiqMoverTest < TransferJobsTestCase
+  include JobHelper
+
+  def setup
+    @source = Sidekiq.redis {|redis| redis }
+    @dest = Redis.new(url: redis_url + "/1")
+  end
+
+  def teardown
+    JobHelper.job_result_queue = []
+    @dest.flushdb
```

Comment:
It probably doesn't matter, but why not flushdb during test setup instead of during tear down. Doing it during setup just ensures that the tests runs with the clean state (probably only makes a difference for the first test in the set anyway)

Reply:
It's just the way we typically handle resources in core... we wipe the slate clean on teardown.